.bit/responses/2.1-Week 2 Step 1.md

```diff
@@ -0,0 +1,103 @@
+---
+files: n/a
+stepType: checks
+scripts: test.parseimage.js
+week: 2
+step: 1
+name: Week 2 Step 1
+---
+
+## Week 2 Step 1
+
+### [Parsing](https://oxylabs.io/blog/what-is-data-parser#:~:text=Data%20parsing%20is%20a%20method,be%20easily%20read%20and%20understood.) Multipart
+
+In your Azure Portal, go into your HTTP function and into the index.js file. You should see starter code there that has something like `module.exports...`.
+This is the file that we will be editing.
+
+**The Azure Function needs to:**
+
+1. Receive and parse an image from a webpage
+2. Call the Face API and analyze the image 
+
+![flowchart](https://user-images.githubusercontent.com/69332964/103604140-3e4ef400-4ede-11eb-8b0e-f3b6b3f203b7.png)
+
+*What is happening in this flowchart?*
+
+**HTML Page:** Where the user will submit an image; this page will also make a request with the image to the Azure Function.
+
+**Azure Function:** Receives the request from HTML Page with the image. In order to receive it, **we must parse the image...**
+
+### Concepts to know:
+* Parsing data
+* Node dependency packages
+* Body and Headers of a request
+
+*We're going to be focusing on Part 1, which involves parsing [multipart form data](https://stackoverflow.com/questions/4526273/what-does-enctype-multipart-form-data-mean).* 
+
+### 📝 Task 1: Analyze image input using the Face API
```

Comment:
Great work with identifying what task they need to complete - now just add a "to move on" section and make the tasks h2, not h3.

format:
## :camping: To move on [insert what to do here]

This is what students would need to do to move on to the next step in the week.